docs/notebooks/ppg/ppg_analysis.ipynb

```diff
@@ -9,23 +9,19 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 1,
```

Comment:
Execution counts are still visible, always difficult ;)

Reply:
I don't know what happened here, since some cells are null while others are not... But now I updated this